lib/fetch/index.js

```diff
@@ -313,16 +313,14 @@ function finalizeAndReportTiming (response, initiatorType = 'other') {
 }
 
 // https://w3c.github.io/resource-timing/#dfn-mark-resource-timing
-function markResourceTiming (timingInfo, originalURL, initiatorType, globalThis, cacheState) {
-  if (nodeMajor > 18 || (nodeMajor === 18 && nodeMinor >= 2)) {
-    performance.markResourceTiming(timingInfo, originalURL.href, initiatorType, globalThis, cacheState)
-  }
-}
+const markResourceTiming = (nodeMajor > 18 || (nodeMajor === 18 && nodeMinor >= 2))
+  ? performance.markResourceTiming
+  : () => {}
 
 // https://fetch.spec.whatwg.org/#abort-fetch
 function abortFetch (p, request, responseObject, error) {
   // 1. Reject promise with error.
-  p.reject(error)
+  setImmediate(() => p.reject(error))
```

Comment:
This shouldn't be necessary according to spec. Are we sure this is a spec bug and not something else?

Reply:
Something is clogging the event loop. As I see other tests fail. So this is not the solution for the issue.

calling setImmediate resulted in declogging it. Still investigating.

Reply:
I think this is an issue in v8, I vaguely recall an issue in node core where promises weren't being GC'ed in a loop

Reply:
https://github.com/nodejs/node/issues/43655